lib/scarpe/wv/image.rb

```diff
@@ -1,16 +1,33 @@
 # frozen_string_literal: true
 
-require "scarpe/components/base64"
+require "scarpe/components/asset_server"
 
 module Scarpe::Webview
-  class Image < Drawable
-    include Scarpe::Components::Base64
+  def self.asset_server
+    return @asset_server if @asset_server
+
+    @asset_server = Scarpe::Components::AssetServer.new
+
+    # at_exit doesn't work reliably under webview. Give this a try.
+    ::Scarpe::Webview::DisplayService.instance.control_interface.on_event(:shutdown) do
+      @asset_server&.kill_server
+    end
+
+    @asset_server
+  end
+end
 
+module Scarpe::Webview
+  class Image < Drawable
     def initialize(properties)
       super
 
       unless valid_url?(@url)
-        @url = "data:image/png;base64,#{encode_file_to_base64(@url)}"
+        # It's assumed to be a file path.
+        @url = File.expand_path @url
+        file_size = File.size(@url)
```

Comment:
Do we need this `file_size`? it seems it's doing the size check inside the asset_server?

Reply:
Oh hey, you're right. That's a leftover from where the code used to live. I'll fix it.